flow/beta/cli/config/json/account_test.go

```diff
@@ -0,0 +1,322 @@
+		}
+	}`)
+
+	var jsonAccounts jsonAccounts
+	json.Unmarshal(b, &jsonAccounts)
+
+	accounts := jsonAccounts.transformToConfig()
+
+	assert.Equal(t, "f8d6e0586b0a20c7", accounts.GetByName("emulator-account").Address.String())
+	assert.Equal(t, "flow-emulator", accounts.GetByName("emulator-account").ChainID.String())
+	assert.Equal(t, 1, len(accounts.GetByName("emulator-account").Keys))
+	assert.Equal(t, "SHA3_256", accounts.GetByName("emulator-account").Keys[0].HashAlgo.String())
+	assert.Equal(t, 0, accounts.GetByName("emulator-account").Keys[0].Index)
+	assert.Equal(t, "ECDSA_P256", accounts.GetByName("emulator-account").Keys[0].SigAlgo.String())
+	assert.Equal(t, "1272967fd2bd75234ae9037dd4694c1f00baad63a10c35172bf65fbb8ad74b47", accounts.GetByName("emulator-account").Keys[0].Context["privateKey"])
+
+	assert.Equal(t, "3c1162386b0a245f", accounts.GetByName("testnet-account").Address.String())
+	assert.Equal(t, "testnet", accounts.GetByName("testnet-account").ChainID.String())
+	assert.Equal(t, 1, len(accounts.GetByName("testnet-account").Keys))
+	assert.Equal(t, "SHA3_256", accounts.GetByName("testnet-account").Keys[0].HashAlgo.String())
+	assert.Equal(t, 0, accounts.GetByName("testnet-account").Keys[0].Index)
+	assert.Equal(t, "ECDSA_P256", accounts.GetByName("testnet-account").Keys[0].SigAlgo.String())
+	assert.Equal(t, "2272967fd2bd75234ae9037dd4694c1f00baad63a10c35172bf65fbb8ad74b47", accounts.GetByName("testnet-account").Keys[0].Context["privateKey"])
+}
+
+func Test_ConfigAccountsMap(t *testing.T) {
+	b := []byte(`{
+		"emulator-account": {
+			"address": "service-1",
+			"chain": "flow-emulator",
+			"keys": "dd72967fd2bd75234ae9037dd4694c1f00baad63a10c35172bf65fbb8ad74b47"
+		},
+		"testnet-account": {
+			"address": "3c1162386b0a245f",
+			"chain": "testnet",
+			"keys": "1232967fd2bd75234ae9037dd4694c1f00baad63a10c35172bf65fbb8ad74b47"
+		}
+	}`)
+
+	var jsonAccounts jsonAccounts
+	json.Unmarshal(b, &jsonAccounts)
+
+	accounts := jsonAccounts.transformToConfig()
+
+	assert.Equal(t, "0000000000000000", accounts.GetByName("emulator-account").Address.String())
+	assert.Equal(t, "emulator-account", accounts.GetByName("emulator-account").Name)
+}
+
+func Test_TransformDefaultAccountToJSON(t *testing.T) {
+	b := []byte(`{"emulator-account":{"address":"f8d6e0586b0a20c7","chain":"flow-emulator","keys":[{"type":"hex","index":0,"signatureAlgorithm":"ECDSA_P256","hashAlgorithm":"SHA3_256","context":{"privateKey":"1272967fd2bd75234ae9037dd4694c1f00baad63a10c35172bf65fbb8ad74b47"}}]},"testnet-account":{"address":"3c1162386b0a245f","keys":"2272967fd2bd75234ae9037dd4694c1f00baad63a10c35172bf65fbb8ad74b47","chain":"testnet"}}`)
+
+	var jsonAccounts jsonAccounts
+	json.Unmarshal(b, &jsonAccounts)
+
+	accounts := jsonAccounts.transformToConfig()
+
+	j := jsonAccounts.transformToJSON(accounts)
+	x, _ := json.Marshal(j)
+
+	// our output format is shorted - improve test
+	assert.NotEqual(t, string(b), string(x))
```

Comment:
You can use `assert.JSONEq` here -- it will normalize the JSON strings and compare for equality.